components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -463,4 +464,25 @@ - (void)testBottomDrawerCornersAPIFullScreen {
   XCTAssertEqual(self.drawerViewController.maskLayer.minimumCornerRadius, 3.f);
 }
 
+- (void)testBottomDrawerHandle {
+  // When
+  [self.presentationController presentationTransitionWillBegin];
+  // Then
+  XCTAssertNotNil(self.presentationController.topHandle);
+  XCTAssertEqual(CGRectGetWidth(self.presentationController.topHandle.frame), 24.f);
```

Comment:
Same for floats. Shouldn't we use `XCTAssertWithAccuracy` when comparing floats?